internal/util/util.go

```diff
@@ -108,13 +108,9 @@ func LabelMapToArray(labels *map[string]string) []string {
 	if labels == nil {
 		return []string{}
 	}
-	output := make([]string, len(*labels))
-	i := 0
-	for key, val := range *labels {
-		output[i] = fmt.Sprintf("%s=%s", key, val)
-		i++
-	}
-	return output
+	return lo.Map(lo.Entries(*labels), func(entry lo.Entry[string, string], _ int) string {
```

Comment:
This is great example for what we discussed on the team call today, where introducing `lo` may have undesirable side-effects besides adding a dependency, which is encouraging changing the programming style.

IMO, this change does not improve readability, conciseness, performance, etc, but on the contrary reduces readability as it assumes
knowledge of `lo` and isn’t aligned with the style used elsewhere in the codebase.

Reply:
reverted